server.js

```diff
@@ -1,16 +1,10 @@
 import express from "express";
 import cors from "cors";
 import mongoose from "mongoose";
+import booksData from "./data/books.json";
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
+// Had to change localhost to 127.0.0.1:27017 to stop constant crashing.
```

Comment:
Glad that you made it work! It's always some little detail that crashes everything.. :-)